### ISSUE TITLE
Problems setting a different default theme in site.yml

### ISSUE THREAD

**Issue by JHSheridan** [vccw-team/vccw#82]
I'm not sure if this is a bug or just my own incompetence, but I'm looking for a little help setting a theme in site.yml.

I'm trying to use this theme: [repo link](https://github.com/digisavvy/some-like-it-neat)

From my understanding of the documentation, all I need to do is change the theme parameter in site.yml to the URL of the .zip of my desired theme, so my site.yml reads: 

```
theme: https://github.com/digisavvy/some-like-it-neat/archive/master.zip
```
I have also tried it as:
```
theme: 'https://github.com/digisavvy/some-like-it-neat/archive/master.zip'
```
I get the following error:
```
==> spwp.dev: FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
==> spwp.dev: ERROR: bash[WordPress https://github.com/digisavvy/some-like-it-neat/archive/master.zip activate] (wpcli::install line 185) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
==> spwp.dev: ---- Begin output of "bash"  "/tmp/chef-script20150718-26805-m5zhk6" ----
==> spwp.dev: STDOUT:
==> spwp.dev: STDERR: Error: The 'master' theme could not be found.
==> spwp.dev: ---- End output of "bash"  "/tmp/chef-script20150718-26805-m5zhk6" ----
==> spwp.dev: Ran "bash"  "/tmp/chef-script20150718-26805-m5zhk6" returned 1
==> spwp.dev: FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
Chef never successfully completed! Any errors should be visible in the
output above. Please fix your recipes so that they properly complete.
```
I'm not sure if maybe linking directly to the github .zip download is an issue or not. Should I try hosting it on a server of my own instead? Or maybe I'm just missing something simple somewhere.

**Comment by miya0001** [comment]
Thanks!
I released 2.12.0.
https://github.com/vccw-team/vccw/releases/tag/2.12.0

**Comment by JHSheridan** [comment]
:+1: 

Awesome. Thanks for the quick response and fix!
